service/web/adminserver.go

```diff
@@ -104,6 +104,17 @@ func (s adminServer) ModifyTopic(ctx context.Context, req *metrov1.AdminTopic) (
 		return nil, merror.ToGRPCError(aerr)
 	}
 
+	existingTopic, eerr := s.topicCore.Get(ctx, m.Name)
```

Comment:
This should not happen before the actual topic has been scaled up.

Reply:
Also please avoid using variables such as eerr, err1, err2, etc. Either give them meaningful names or re-use `err`